articles/logs/streams/datadog.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Datadog Event Log Streams
+description: Learn about Datadog Event Log Streams and how they let you export your events in near real-time to Datadog.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Datadog Event Log Streams
+
+Datadog is the essential monitoring platform for cloud applications. It brings together data from servers, containers, databases, and third-party services to make your stack entirely observable.
+
+You can create monitoring, alerting, and analysis dashboards for Auth0 tenants.
+
+## Send events from Auth0 to Datadog
+
+To send Auth0 events to Datadog, you will need:
+
+1. A Datadog API Key.
+2. Your Datadog dashboard region.
+
+### Copy API Key from Datadog
+
+1. Log in to the Datadog dashboard.
+2. Navigate to **Integrations** > **APIs**.
+![Integrations Dashboard](/media/articles/logs/datadog/tutorial-1.png)
+3. Expand the API Keys section, and copy the API Key that you would like to use.
+![API Keys Section](/media/articles/logs/datadog/tutorial-2.png)
+
+### Set up Event Stream in Auth0
+
+1. Log in to the [Auth0 Dashboard](${manage_url}).
+2. Navigate to **Logs > Streams**.
+3. Click **+ Create Stream**.
+4. Select **Datadog**, and enter a unique name for your new Datadog Event Stream.
+5. On the next screen, provide the following settings for your Datadog Event Stream:
+
+| Setting | Description |
+|---------|-------------|
+| API Key | The Datadog API key you copied from the Datadog dashboard. |
+| Region  | If you are in the Datadog EU site (app.datadoghq.eu), the `Region` should be `EU`; otherwise, it should be `US`. |
+
+![Datadog Settings Form](/media/articles/logs/datadog/tutorial-3.png)
+
+6. Click **Save**.
+7. You're done! When Auth0 writes the next tenant log, you'll receive a copy of that log event in Datadog with the `source` and `service` set to `auth0`.
```

Comment:
I would rename "tenant log" to something else. From our point of view these are tenants and their tenant logs (as opposed to application logs).

But as a customer these are just logs or log events.

What about `When Auth0 writes the next log event`

Reply:
That's an excellent point. I've updated the docs accordingly. Thank you for your feedback!